### ISSUE TITLE
Can't play when run locally

### ISSUE THREAD

**Issue by 4rch1m3d35** [inolen/quakejs#97]
I cloned and ran this according to the instructions and I can't get past the opening menu. Is there content I haven't downloaded somewhere? or is this just not working anymore?

**Comment by 4rch1m3d35** [comment]
I'm running off of the Windows Linux Subsystem on Windows 11

**Comment by saranshgupta0107** [comment]
agreed after starting the server, when i open 8080 then i can see only background picture and game is not loading

**Comment by crysre** [comment]
When I run this command-
 echo '{ "content": "content.quakejs.com" }' > bin/web.json command 

i get this error- 
bash: bin/web.json: No such file or directory

**Comment by Sunnahboy** [comment]
when i run i get this,
initial sync completed in 0.03 seconds
localhost/:1  Access to XMLHttpRequest at 'https://content.quakejs.com/assets/manifest.json' (redirected from 'http://content.quakejs.com/assets/manifest.json') from origin 'http://localhost:8080' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
content.quakejs.com/assets/manifest.json:1 
        
        
        Failed to load resource: net::ERR_FAILED

**Comment by AsishDatta** [comment]
I am getting this error, when I run locally. 
 loading config file from ./web.json..
warn:    failed to load config SyntaxError: C:\Users\asish\OneDrive\Desktop\Web Development\verson control\quakejs\bin\web.json: Unexpected token '�', "��{ "c"... is not valid JSON
    at parse (<anonymous>)
    at Module._extensions..json (node:internal/modules/cjs/loader:1434:39)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Module.require (node:internal/modules/cjs/loader:1233:19)
    at require (node:internal/modules/helpers:179:18)
    at loadConfig (C:\Users\asish\OneDrive\Desktop\Web Development\verson control\quakejs\bin\web.js:30:14)
    at Object.<anonymous> (C:\Users\asish\OneDrive\Desktop\Web Development\verson control\quakejs\bin\web.js:20:14)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)

**Comment by AsishDatta** [comment]
info:    loading config file from ./web.json..
warn:    failed to load config SyntaxError:
\quakejs\bin\web.json: Unexpected token '�', "��{ "c"... is not valid JSON
    at parse (<anonymous>)
    at Module._extensions..json (node:internal/modules/cjs/loader:1434:39)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Module.require (node:internal/modules/cjs/loader:1233:19)
    at require (node:internal/modules/helpers:179:18)

**Comment by Lokesh1933** [comment]
Even after running npm audit fix --force the server is not accessible through localhost
